cmd/kubeadm/app/apis/kubeadm/bootstraptokenhelpers.go

```diff
@@ -26,16 +26,16 @@ import (
 	"k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	bootstrapapi "k8s.io/cluster-bootstrap/token/api"
-	bootstraputil "k8s.io/cluster-bootstrap/token/util"
+	bootstraptokenutil "k8s.io/cluster-bootstrap/util/tokens"
```

Comment:
my preference would be to move this the other way around....

cluster-bootstrap/util/tokens/tokens.go has a single function:
`ParseToken()`

move that to:
`cluster-bootstrap/token/util/helpers.go`

and add a unit test for it in helpers_test.go.